### PR TITLE
add-address-form

### PR DESCRIPTION
**Pull request by viniciusreiss**
close adicionar endereco no cadastro de contrato #82

### CODE REVIEW

File: src/pages/Form/index.js [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
+  state = {
+    specialties: [],
   }
 
-  getSpecialties = () => { 
+  getAddress = async (e) => { 
+    const zipCode = e.target.value
+    try {
+      const address = await getAddressByZipCode(zipCode)
+      console.log(address)
```
Comment (vitorsilvalima) [inline src/pages/Form/index.js]: remover console.log

File: src/pages/index.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ class App extends Component {
         <Route path="/loading" component={Loading} />
         <Route path="/SucessMessage"  component={SucessMessage} />
         <Route path="/ErrorMessage"  component={ErrorMessage} />
-        <Redirect to="/welcome?token=58fe73b5-b3e8-4b38-b633-badb3eb3413f&name=Andre&email=andre1@gmail.com"  />
+        <Redirect to="/welcome?token=0679dea2-f7db-43ee-b526-d090d646401bf&name=Batata&email=batatinha@gmail.com"  />
```
Comment (vitorsilvalima) [inline src/pages/index.js]: removed token

File: src/pages/Confirmation/index.js [hunk 3]
```diff
@@ -12,12 +12,23 @@ class Confirmation extends Component {
...
+      neighborhood: body.neighborhood,
+      number: body.number,
+      complement: body.complement,
+      zipCode: body.zipCode
+    }
     const newBody = {
       ...body,
+      ...address,
       cpf: body.cpf.replace(/\.|\-/g, ''),
       cellPhone: body.cellPhone.replace(/([\()-])/g, ''),
```
Comment (alexandrejuk) [inline src/pages/Confirmation/index.js]: this ".replace()" should be a function

File: src/pages/Confirmation/index.js [hunk 4]
```diff
@@ -12,12 +12,23 @@ class Confirmation extends Component {
...
       zipCode: body.zipCode.replace(/([\()-])/g, '')
     }    
+    console.log(newBody)
```
Comment (alexandrejuk) [inline src/pages/Confirmation/index.js]: remove this console.log

File: src/pages/Confirmation/index.js [hunk 5]
```diff
@@ -1,30 +1,44 @@
...
+import * as R from 'ramda'
 class Confirmation extends Component {
 
   state = {
     loading: false
   }
 
-  postData = () => {
-    const token = localStorage.getItem('token')
+  getBodyAndFormat = () => {
```
Comment (vitorsilvalima) [inline src/pages/Confirmation/index.js]: getFormattedContractor

**Pull request merged**